images/linux/Ubuntu2004-README.md

```diff
@@ -119,42 +118,42 @@
 | 11.0.11+9 (default) | Adopt OpenJDK | JAVA_HOME_11_X64     |
 
 ### GraalVM
-| Version   | Environment variables |
-| --------- | --------------------- |
-| CE 21.2.0 | GRAALVM_11_ROOT       |
+| Version     | Environment variables |
+| ----------- | --------------------- |
+| CE 21.0.0.2 | GRAALVM_11_ROOT       |
```

Comment:
Is it downgraded?

Reply:
@miketimofeev something happened and now 21.0.0.2 version is in latest release